.vscode/settings.json

```diff
@@ -1,8 +1,8 @@
 {
   "css.lint.validProperties": ["composes"],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.fixAll.shellcheck": true
+    "source.fixAll.eslint": "explicit",
+    "source.fixAll.shellcheck": "explicit"
```

Comment:
I'm not sure if this was intended, mind switching it back to `true`

Reply:
thanks! that wasn't intended. not sure how it happened. yes, will switch back. good catch on the segmented control. wonder if changing to the "small" variant will fix.

Reply:
@rlingineni i think I fixed it

Reply:
Hey, these are still showing up as "explicit"

I think you can run a command like this to reset a file back to how it is in master
```
git checkout master -- .vscode/settings.json
```